doc/api/Renter.md

```diff
@@ -212,6 +212,12 @@ lists the status of all files.
       // with 0 redundancy.
       "redundancy": 5,
 
+      // Total number of bytes successfully uploaded via current file contracts.
+      // This number includes padding and rendundancy, so a file with a size of
+      // 8192 bytes might be padded to 40 MiB and, with a redundancy of 5,
+      // striped to 200 MiB for upload.
```

Comment:
important nitpick - 'striped' is the wrong word here, need to use the word 'encoded'. The redundancy scheme that we use is far superior to striping, and I don't want anyone reading the docs to think we did our redundancy in a naive way.